src/MusicalScore/Graphical/VexFlow/VexFlowMeasure.ts

```diff
@@ -1383,15 +1386,6 @@ export class VexFlowMeasure extends GraphicalMeasure {
             }
             fingeringIndex++; // 0 for first fingering
             let fingeringPosition: PlacementEnum = this.rules.FingeringPosition;
-            //currently only relevant for grace notes, because we create other fingerings above/below in MusicSheetCalculator.createFingerings
-            if (this.rules.FingeringPositionGrace === PlacementEnum.AboveOrBelow) {
-                //if (this.rules.FingeringPosition === PlacementEnum.AboveOrBelow) {
-                if (this.isUpperStaffOfInstrument()) { // (e.g. piano right hand)
-                    fingeringPosition = PlacementEnum.Above;
-                } else if (this.isLowerStaffOfInstrument()) {
-                    fingeringPosition = PlacementEnum.Below;
-                }
-            }
```

Comment:
Revert this change, I do not see the point.

Reply:
Yes. In general, you shouldn't change OSMD's logic in this PR, only how it interfaces with Vexflow.

Reply:
Done